### ISSUE TITLE
Yes/No Boolean Question - A screen reader doesn't announce the Yes/No Options

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7691]
User Issue: T16404 - Accessibility/screen reader issues
https://surveyjs.answerdesk.io/internal/ticket/details/T16404

**Comment by dmitrykurmanov** [comment]
https://www.accessibility-developer-guide.com/knowledge/screen-readers/desktop/browse-focus-modes/
https://github.com/nvaccess/nvda/issues/11143
